src/containers/Search/CloneModel.js

```diff
@@ -0,0 +1,160 @@
+                      } else {
+                        setActiveProject(counter_top + 1);
+                        setCurrentProject(data.id);
+                        setCurrentPlaylist();
+                        // setactiveactivity();
+                      }
+                      // setActivePlaylist();
+                    }}
+                  >
+                    <div className="flex-bar">
+                      <div>
+                        {activeProject === counter_top + 1 ? (
+
+                          <FontAwesomeIcon icon="check-square" />
+                        ) : (
+                            <FontAwesomeIcon icon="square" />
+                        )}
+                        {data.name}
+                      </div>
+                      {activeProject === counter_top + 1 ? (
+                        <FontAwesomeIcon icon="chevron-up" />
+                      ) : (
+                        <FontAwesomeIcon icon="chevron-down" />
+                      )}
+                    </div>
+                  </span>
+                </Accordion.Toggle>
+              </Card.Header>
+              <Accordion.Collapse eventKey={counter_top + 1}>
+                <Card.Body>
+                  {!!data.playlists && data.playlists.length > 0 ? (
+                    <Accordion>
+                      {data.playlists.map((data2, counter_playlist) => (
+                        <Card>
+                          <Card.Header className="middlecard">
+                              <Accordion.Toggle
+                                as={Button}
+                                variant="link"
+                                eventKey={counter_playlist + counter_top + 1}
+                              >
+                                <span
+                                  onClick={() => {
+                                    if (
+                                      activePlaylist
+                                        === counter_playlist + counter_top + 1
+                                    ) {
+                                      setActivePlaylist();
+                                    } else {
+                                      setActivePlaylist(counter_playlist + counter_top + 1);
+
+                                      setCurrentPlaylist(data2.id);
+                                    }
+                                  }}
+                                >
+                                  <div className="flex-bar">
+                                    <div>
+                                      <span>
+
+                                          {activePlaylist
+                                          === counter_playlist + counter_top + 1 ? (
+
+                                            <FontAwesomeIcon icon="stop-circle" />
+                                            ) : (
+                                              <FontAwesomeIcon icon="circle" />
+                                            )}
```

Comment:
Even though we are running out of time, can you improve the code quality?
I am sure that you will not need more time to do that

Reply:
done